otx/algorithms/classification/configs/base/models/efficientnet.py

```diff
@@ -0,0 +1,14 @@
+"""Base EfficientNet."""
+
+# model settings
+model = dict(
+    type="ImageClassifier",
+    backbone=dict(type="OTEEfficientNet", pretrained=True, version="b0"),
```

Comment:
Can't we get this (Backbone, OTEEfficientNet)  in OTX?

Reply:
I can. But didn't detection just configure the backbone, not brining the backbone files to otx? But anyway I agree that it quite irritates me because of 'OTE~'.

Reply:
Detection imported all backbone configuration files into otx.

Reply:
@harimkang Could you check this part and adapters/mmcls/models again, while I go over additional pre-commit tests?